### DISCUSSION TITLE
Frsky  Horus10s with IRSM board

### DISCUSSION THREAD

**Discussion by alant101** [EdgeTX/edgetx#2099] [Q&A] [answered]
### Is there an existing issue for this feature request?

- [X] I have searched the existing issues

### Is your feature request related to a problem?

No available at present

### Describe the solution you'd like

There seems to be various builds of Edgetx noheli available, but none for the Horus 10 with the replacements IRSM board for ACCESS. This would be appreciated if possible

### Describe alternatives you've considered

None

### Additional context

None

**Comment by raphaelcoeffic** [comment]
Please use the discussion related to ask for specific builds ;-)

**Comment by pfeerick** [comment, marked as answer]
Use the X10 build (normal, noheli, etc), and use Radio Settings -> Hardware to set the Internal RF type to ISRM. 
As mentioned / screenshot shown here: https://github.com/EdgeTX/edgetx/discussions/1951#discussioncomment-2851833

Then when you go back to your model settings you'll be able to enable internal RF as ISRM

**Reply by alant101** [reply]
Thanks..........
